cumulus/tasks/sf-sns-report/tests/index.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const test = require('ava');
+const { recursivelyDeleteS3Bucket, s3 } = require('@cumulus/common/aws');
+const { publishSnsMessage } = require('../index');
+const { cloneDeep } = require('lodash');
+const { randomString } = require('@cumulus/common/test-utils');
+
+test('send report when sfn is running', (t) => {
+  const event = {
+    input: {
+      meta: { topic_arn: 'test_topic_arn' },
+      anykey: 'anyvalue'
+    }
+  };
+
+  return publishSnsMessage(cloneDeep(event))
+    .then((output) => {
+      event.input.meta.status = 'running';
+      t.deepEqual(output, event.input);
+    });
+});
+
+test('send report when sfn is running with exception', (t) => {
+  const event = {
+    input: {
+      meta: { topic_arn: 'test_topic_arn' },
+      exception: {
+        Error: 'TheError',
+        Cause: 'bucket not found'
+      },
+      anykey: 'anyvalue'
+    }
+  };
+
+  return publishSnsMessage(cloneDeep(event))
+    .catch((e) => {
+      t.is(e.message, event.input.exception.Cause);
+    });
+});
+
+test('send report when sfn is running with error', (t) => {
+  const event = {
+    input: {
+      meta: { topic_arn: 'test_topic_arn' },
+      error: {
+        Error: 'TypeError',
+        Cause: 'bucket not found'
+      },
+      anykey: 'anyvalue'
+    }
+  };
+
+  return publishSnsMessage(cloneDeep(event))
+    .catch((e) => {
+      t.is(e.message, event.input.error.Cause);
+    });
+});
+
+
+test('send report when sfn is finished and granule succeed', async (t) => {
```

Comment:
`send report when sfn is finished and granule has succeeded`

Reply:
done